sample-scenarios/sample7/src/main/java/CreateSampleSevenRawData.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.wso2.carbon.apimgt.samples.utils.SampleUtils;
+import org.wso2.carbon.apimgt.samples.utils.publisher.rest.client.ApiException;
+import org.wso2.carbon.apimgt.samples.utils.publisher.rest.client.model.API;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * This class is used to populate sample data to represent the sample Seven business scenario.
+ */
+public class CreateSampleSevenRawData {
```

Comment:
Can we add a comment saying this is for the sample business scenario "API Lifecycle Management"  would be easy for later. Also renaming the class with that also if possible?

Reply:
Thank you for pointing out. I'll fix and update